Automattic/woocommerce-payments
# Add badge to Payments > Settings if UPE settings preview = 1 and UPE = 0

Fixes #2173

#### Changes proposed in this Pull Request

When the UPE settings preview feature flag is enabled and the UPE flag isn't, a badge is displayed next to Payments > Settings.









#### Testing instructions





* Enable `_wcpay_feature_upe_settings_preview` and disable `_wcpay_feature_upe`.
    * The badge is displayed.

    ![Markup on 2021-06-18 at 20:02:09](https://user-images.githubusercontent.com/1759681/122600567-41930200-d070-11eb-9892-154e4e20c6aa.png)

* Toggle any or both of the above.
    * The badge is not displayed.

    ![Markup on 2021-06-18 at 20:03:08](https://user-images.githubusercontent.com/1759681/122600576-45268900-d070-11eb-8e8a-437efd280f11.png)




-------------------

    - Not added - unreleased feature.
